src/CoreBundle/Controller/ExceptionController.php

```diff
@@ -73,7 +75,9 @@ public function setMainController(ChameleonControllerInterface $mainController)
      */
     public function showAction(Request $request, FlattenException $exception, DebugLoggerInterface $logger = null)
     {
-        $currentContent = $this->getAndCleanOutputBuffering($request->headers->get('X-Php-Ob-Level', -1));
+        /** @var int $level */
+        $level = $request->headers->get('X-Php-Ob-Level', -1);
```

Comment:
I extracted this into a variable so I get to declare a type that is 'more narrow' than the return type of `$request->headers->get`.